portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,56 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Anooj Shah's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Caveat&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="content" class="hover01">
+      <h1>Anooj Shah's Portfolio</h1>
+      <figure> <img src="./images/anoojshah.png" id="profile" alt="Anooj Profile Picture"> </figure>
+      <p>My name is Anooj! I am a Sophomore Computer Science Major at Texas A&M University.</p>
+      <!-- <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div> -->
+    </div>
+    <div id="Hobbies">
+      <h2>Here are some of my Interests</h2>
+      <div id='hobby'>
```

Comment:
From a consistency standpoint, it would be preferable to standardize on either single or double quotes. This consistency reduces the cognitive load placed on readers trying to understand your changes. I don't feel strongly, but I think double quotes looks a bit better.

(A word about my review style: this comment applies to many lines in the PR. When I make comments that discuss principles that apply widely, the hope is that you will make changes as appropriate throughout the PR.)